### PR TITLE
CSS of Conjugator

### PR DESCRIPTION
**Pull request by bettychow**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -2,34 +2,61 @@ import React, { Component } from 'react';
...
+      <label className="button">
+        <input type="radio" className="button"/> Option 6
+      </label>
+    </div>
 
     <div>
-      <button type="button" className="btn btn-primary" onChange={() => {
-        submit();
-      }}>Submit</button>
+      <img src={arrow} width="73" height="auto" onClick={ () => { submit(); } } />
```
Comment (songz) [inline src/components/App.js]: `;` - put image in `public` folder

File: src/style/App.css [hunk 2]
```diff
@@ -48,4 +49,53 @@ pre code {
   padding: 8px 0;
 }
 
-  
+
+.title-container {
+    margin-top: 40px;
```
Comment (songz) [inline src/style/App.css]: top/bottom/left/right can be combined into - `margin: 40px auto;`

File: src/style/App.css [hunk 3]
```diff
@@ -48,4 +49,53 @@ pre code {
...
+    margin-bottom: 40px;
+    margin-left: auto;
+    margin-right: auto;
+    color: #ffffff;
+    padding-top: 20px;
```
Comment (songz) [inline src/style/App.css]: y padding top here?

File: src/components/App.js [hunk 4]
```diff
@@ -2,34 +2,61 @@ import React, { Component } from 'react';
...
 
   return (
+
     <div className="App">
-      <div>
-        <h1>Select Your Language!</h1>
+      <div className="title-container">
+              <p className="title" >CONJUGATOR</p>
       </div>
+        <p className="text1">Select Your Language!</p>
```
Comment (songz) [inline src/components/App.js]: indent

File: src/components/App.js [hunk 5]
```diff
@@ -2,34 +2,61 @@ import React, { Component } from 'react';
...
 const App = ({word, selectedLanguage, changeLanguage, submit, changeWord}) => {
   const languageList = ['Korean', 'French', 'Hindi'];
 
   return (
+
     <div className="App">
-      <div>
-        <h1>Select Your Language!</h1>
+      <div className="title-container">
+              <p className="title" >CONJUGATOR</p>
```
Comment (songz) [inline src/components/App.js]: indent

File: src/components/App.js [hunk 6]
```diff
@@ -2,34 +2,61 @@ import React, { Component } from 'react';
...
-      
-    <Dropdown options={languageList} onChange={changeLanguage} value={selectedLanguage} placeholder="Please Select Language" />
 
-      <div>
+    <Dropdown options={languageList} onChange={changeLanguage} value={selectedLanguage} placeholder="Please Select Language" />
+    <p className="text1">Type Your Word</p>
+    <div>
         <form>
-          <input type="text" name="word" placeholder="Please input word" value={word} onChange={(e) => {
+          <input type="text" className="box1" name="word" value={word} onChange={(e) => {
```
Comment (songz) [inline src/components/App.js]: classname should be more descriptive

File: src/style/App.css [hunk 7]
```diff
@@ -48,4 +49,53 @@ pre code {
...
+}
+.title {
+   display: inline;
```
Comment (songz) [inline src/style/App.css]: if display inline, maybe change to `span`?

File: src/style/ReactDropdown.css [hunk 8]
```diff
@@ -1,3 +1,32 @@
+<<<<<<< Updated upstream
```
Comment (songz) [inline src/style/ReactDropdown.css]: merge conflict?

File: src/style/ReactDropdown.css [hunk 9]
```diff
@@ -1,3 +1,32 @@
...
+
+pre code {
+  line-height: 20px;
+}
+
+.result {
+  padding: 8px 0;
+}
+
+>>>>>>> Stashed changes
```
Comment (songz) [inline src/style/ReactDropdown.css]: merge conflict?

File: src/style/App.css [hunk 10]
```diff
@@ -48,4 +49,53 @@ pre code {
...
+   padding-left: 20px;
+   padding-right: 20px;
+   font-size: 3em;
+   font-family: 'Archivo Black', sans-serif;
+   color: #ffffff;$
+   font-weight: bolder;
+}
+.text1 {
+    color: #ffffff;
+      font-size: 18px;
```
Comment (songz) [inline src/style/App.css]: indent

**Comment by songz** [comment]
delete asset image

File: package.json [hunk 11]
```diff
@@ -6,6 +6,7 @@
     "react": "^15.5.4",
     "react-dom": "^15.5.4",
     "react-dropdown": "^1.2.3",
+    "react-image": "^1.0.1",
```
Comment (songz) [inline package.json]: delete

File: src/components/App.js [hunk 12]
```diff
@@ -2,34 +2,65 @@ import React, { Component } from 'react';
 import '../style/App.css';
 import '../style/ReactDropdown.css';
 import Dropdown from 'react-dropdown';
+import Img from 'react-image';
```
Comment (songz) [inline src/components/App.js]: why are you using this?

File: src/style/ReactDropdown.css [hunk 13]
```diff
@@ -49,8 +81,10 @@
   overflow-y: auto;
   position: absolute;
   top: 100%;
-  width: 100%;
-  z-index: 1000;
+  width: 280px;
+  margin-left: 500px;
+  border-radius: 5px;
+  z-index: 1000;;
```
Comment (songz) [inline src/style/ReactDropdown.css]: never use 1000

File: src/style/ReactDropdown.css [hunk 14]
```diff
@@ -1,3 +1,29 @@
+body {
```
Comment (songz) [inline src/style/ReactDropdown.css]: keep this file specifically for styling ReactDropdown component.

File: src/style/ReactDropdown.css [hunk 15]
```diff
@@ -14,6 +40,12 @@
   outline: none;
   padding: 8px 52px 8px 10px;
   transition: all 200ms ease;
+  width: 280px;
```
Comment (songz) [inline src/style/ReactDropdown.css]: don't modify CSS of libraries.

File: src/style/App.css [hunk 16]
```diff
@@ -48,4 +50,49 @@ pre code {
   padding: 8px 0;
 }
 
-  
+.title-container {
+  margin: 40px auto;
```
Comment (songz) [inline src/style/App.css]: this should not have margin auto.

File: src/style/App.css [hunk 17]
```diff
@@ -1,5 +1,33 @@
...
+  line-height: 20px;
+}
+
+.result {
+  padding: 8px 0;
+}
+
 .App {
   text-align: center;
+  background-color: #32bdf9;
```
Comment (songz) [inline src/style/App.css]: why does app have background color?

File: src/style/App.css [hunk 18]
```diff
@@ -48,4 +71,55 @@ pre code {
...
+}
+
+.button {
+  margin-left: 10px;
+  margin-right: 10px
+}
+
+.input-box {
+  border-radius: 5px;
+  width: 280px;
```
Comment (songz) [inline src/style/App.css]: put box on everything, then you only need to specify width once.

File: src/style/App.css [hunk 19]
```diff
@@ -48,4 +71,55 @@ pre code {
...
+}
+
+.output-box {
+  margin-top: 10px;
+  border-radius: 5px;
+  height: 70px;
+  width: 200px;
+} 
+
+.dropdown {
```
Comment (songz) [inline src/style/App.css]: put box on everything, then you only need to specify width once.

File: src/style/ReactDropdown.css [hunk 20]
```diff
@@ -50,7 +50,7 @@
   position: absolute;
   top: 100%;
   width: 100%;
-  z-index: 1000;
+  z-index: 1000;;
```
Comment (songz) [inline src/style/ReactDropdown.css]: one semicolon

File: src/style/ReactDropdown.css [hunk 21]
```diff
@@ -67,6 +67,8 @@
   cursor: pointer;
   display: block;
   padding: 8px 10px;
+  margin-right: auto;
```
Comment (songz) [inline src/style/ReactDropdown.css]: why is this necessary?

File: src/components/App.js [hunk 22]
```diff
@@ -7,29 +7,59 @@ const App = ({word, selectedLanguage, changeLanguage, submit, changeWord}) => {
...
         <form>
-          <input type="text" name="word" placeholder="Please input word" value={word} onChange={(e) => {
+          <input type="text" className="input-box" name="word" value={word} onChange={(e) => {
             changeWord(e.target.value);
           }}/>
-      </form>
+        </form>
+      </div>
+     
+     <div className="radio text2">
```
Comment (songz) [inline src/components/App.js]: avoid using names like text1 or text2. Maybe option-font

File: src/style/App.css [hunk 23]
```diff
@@ -1,5 +1,16 @@
+body {$
+  margin: 20px auto;
```
Comment (songz) [inline src/style/App.css]: why do you need this?

File: src/style/App.css [hunk 24]
```diff
@@ -23,29 +34,55 @@
   to { transform: rotate(360deg); }
 }
 
-body {
-  margin: 20px auto;
-  width: 60%;
+.title-container {
+  margin: 40px auto;
```
Comment (songz) [inline src/style/App.css]: why do you need to auto margin?

File: src/components/App.js [hunk 25]
```diff
@@ -7,11 +7,11 @@ const RadioButtonOptions = ({options, setOptions, title}) => {
   let radioOptions = [];
```
Comment (bettychow) [inline src/components/App.js:7]: screen shot 2017-05-22 at 5 36 15 pm

File: src/components/App.js [hunk 26]
```diff
@@ -25,26 +25,29 @@ const App = ({options, selectedLanguage, changeLanguage, word, changeWord, langu
...
+        <span className="title" >CONJUGATOR</span>
       </div>
-
+      <div className='content-container'>
+        <p className="instruction_text">Select Your Language!</p>
       <Dropdown options={languageList} onChange={(languageOption) => {changeLanguage(languageOption.value)}} value={selectedLanguage} placeholder="Please Select Language" />
       {optionListForm} {optionBulletForm}
-      <div>
-        <input type="text" name="word" placeholder="Please input word" value={word} onChange={(e) => {
+        <p className="instruction_text">Type Your Word</p>
```
Comment (songz) [inline src/components/App.js]: be consistent with naming convention. use dashes instead of underscore: `instruction-text`

File: src/components/App.js [hunk 27]
```diff
@@ -25,26 +25,29 @@ const App = ({options, selectedLanguage, changeLanguage, word, changeWord, langu
...
       {optionListForm} {optionBulletForm}
-      <div>
-        <input type="text" name="word" placeholder="Please input word" value={word} onChange={(e) => {
+        <p className="instruction_text">Type Your Word</p>    
+    <input type="text" className="input-box" name="word" placeholder="Please input word" value={word} onChange={(e) => {
           changeWord(e.target.value);
         }} />
-    </div>
+        <img className="image" src="/arrow_button_white.png"
+          width="73" height="auto" onClick={ () => { submit(); } } />
```
Comment (songz) [inline src/components/App.js]: `onClick={ submit }`

File: src/style/App.css [hunk 28]
```diff
@@ -23,29 +34,73 @@
...
-  border: 1px solid #ccc;
+.instruction_text{
+  margin-top: 30px;
+  font-size: 18px;
+  font-family: ProximaNova-Regular;
 }
 
-pre code {
-  line-height: 20px;
+.option_text {
```
Comment (songz) [inline src/style/App.css]: dashes

File: src/style/App.css [hunk 29]
```diff
@@ -23,29 +33,91 @@
...
 }
 
-pre {
-  background: #f7f7f7;
-  padding: 10px 20px;
-  border-radius: 2px;
-  border: 1px solid #ccc;
+.output-box {
+  border-radius: 5px;
+  width: 280px;
```
Comment (songz) [inline src/style/App.css]: width: 100% to fill the parent. Should hardcode as little as you can.

File: src/style/App.css (hunk 18 above)
Comment (songz) [reply src/style/App.css]: or width: 100%?

File: src/style/App.css [hunk 30]
```diff
@@ -1,5 +1,15 @@
+body {$
+  width: 100%;
```
Comment (songz) [inline src/style/App.css]: isn't body by default width 100% already?

File: src/components/App.js [hunk 31]
```diff
@@ -7,44 +7,47 @@ const RadioButtonOptions = ({options, setOptions, title}) => {
   let radioOptions = [];
     options.map(element => {
       radioOptions.push(
-        <label key={element}>
+        <label className="button" key={element}>
           <input type="radio" name={title} value={element} onChange={() => {setOptions(title, element)}}/>{`${element}`}
         </label>)
     })
-  return (<div>{title}{radioOptions}</div>);
+  return (<div className="radio"><div className="radio-title">{title}</div><span>{radioOptions}</span></div>);
```
Comment (songz) [inline src/components/App.js]: `radio-group` is a better name here.

**Review by songz — APPROVED** [review]
